Sources/ScrollingTabController.swift

```diff
@@ -202,18 +196,32 @@ open class ScrollingTabController: UIViewController, UIScrollViewDelegate, UICol
     }
 
     func configureViewControllers() {
-        for item in items {
```

Comment:
hmm. yea you're right you wouldn't be able to change the initial view controller list here, so maybe we can leave that in. If so it should be noted that you can't change initial view controllers after `viewDidLoad`, otherwise it would probably break stuff.

Also, if we want to go that direction. I don't think the `items` array was getting emptied before, so I'm not sure resetting the list of view controllers would have ever worked?